torch/lib/c10d/reducer.cpp

```diff
@@ -1035,6 +1051,18 @@ void Reducer::copy_bucket_to_grad(
   });
 }
 
+std::vector<std::string> Reducer::getUnmarkedParamsForIteration() {
```

Comment:
could we give both parameter index and name info? some time users can not locate the parameter without knowing the index

Reply:
Yep, sounds good